torchrl/envs/transforms/transforms.py

```diff
@@ -358,21 +344,12 @@ def observation_spec(self) -> TensorSpec:
 
     @property
     def action_spec(self) -> TensorSpec:
-        """Action spec of the transformed_in environment"""
-
-        if self._action_spec is None or not self.cache_specs:
-            action_spec = self.transform.transform_action_spec(
-                deepcopy(self.base_env.action_spec)
-            )
-            if self.cache_specs:
-                self._action_spec = action_spec
-        else:
-            action_spec = self._action_spec
-        return action_spec
+        """Action spec of the transformed environment"""
```

Comment:
Do you think we still need this, as TransformedEnv inherits from EnvBase I guess the end result will be the same

Reply:
If the logic with cache_specs is not mandatory, we could indeed removing it. As a side not this property as no setter so, I am not mistaken, action_spec cannot be be set in a TranformedEnv. (I don't think the setter is inherited if the property is redefined in the child class)